app/models/concerns/radbear_user.rb

```diff
@@ -50,7 +50,7 @@ def firebase_device_tokens(app)
     end
 
     if response.body && response.body.count != 0
-      response.body || []
+      response.body
```

Comment:
Found this while messing with the callbacks. That statement can never return an empty array on that line because you've already confirmed the presence of `response.body`